package.json

```diff
@@ -15,13 +15,16 @@
   "author": "Daniel Siwek",
   "license": "MIT",
   "dependencies": {
+    "chai": "^4.1.2",
     "colors": "^1.3.2",
     "commander": "^2.18.0",
     "fs-extra": "^7.0.0",
-    "replace": "^1.0.0"
+    "mocha": "^5.2.0",
+    "replace": "^1.0.0",
+    "sinon": "^6.3.4"
```

Comment:
shouldn't we use exact version of packages?

Reply:
Solved in #10